include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -1967,27 +2291,71 @@ namespace internal
 
 
 
-  template <int dim,
-            int fe_degree,
-            int n_q_points_1d,
-            typename Number,
-            typename VectorizedArrayType,
-            typename Number2 = Number>
-  struct FEFaceEvaluationSelector
+  template <typename Number>
+  void
+  adjust_for_face_orientation(const unsigned int            dim,
```

Comment:
Does it hurt if you make dim a template parameter?

Reply:
It's just a bit more code to be generated without any benefit - `dim` is only used for an outer loop.